packages/components/src/listbox/src/Listbox.tsx

```diff
@@ -365,8 +365,10 @@ export function InnerListbox({
     });
 
     const scrollableProps = useScrollableCollection(containerRef, nodes, {
+        borderHeight: 2,
```

Comment:
how are these used?

Reply:
The new design adds a border and padding to the list, which the previous one didn't have so we need to add then when calculating the scroll height.